app/src/editor/canvas/components/Toolbar/index.jsx

```diff
@@ -169,8 +170,8 @@ export default withErrorBoundary(ErrorComponentView)(class CanvasToolbar extends
             >
                 <ModalContainer modalId="ShareDialog">
                     {({ api: shareDialog }) => (
-                        <div className={styles.ToolbarInner}>
-                            <div className={styles.LeftControls}>
```

Comment:
`LeftControls` is gone. In this file I pretty much only wrap things with `ToolbarInner.classNames.LEFT/RIGHT/CENTER` divs. FYI.